.github/workflows/dev.yml

```diff
@@ -21,12 +21,11 @@ on:
 jobs:
   check_deps:
     strategy:
+      fail-fast: false
       matrix:
-        python-versions:
-        - "3.10"
-        os:
-        - ubuntu-20.04
-    runs-on: ubuntu-20.04
+        python-versions: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
wont work on 3.8, please dont change the tests hahaha

Reply:
reverted in 64ffd5c3e9fcd9846dc14a6c78e1649a3fd72678
removed mention of python 3.8 from pyproject.toml in f0adb4defbb8944b175793654bcdd5e79cd4235c